tests/test_integrated_channels/test_sap_success_factors/test_transmitters/test_learner_data.py

```diff
@@ -54,7 +54,8 @@ def setUp(self):
                 error_message='',
                 status=200,
                 enterprise_customer_uuid=self.enterprise_customer.uuid,
-                plugin_configuration_id=self.enterprise_config.id
+                plugin_configuration_id=self.enterprise_config.id,
```

Comment:
can we make these changes in `test_transmit_success` and `test_transmit_failure` tests at the place of error_messae and status?

Reply:
@zamanafzal is this what you meant?

https://github.com/openedx/edx-enterprise/pull/2121/files#diff-d45bd0ecb65fe9b632f0d331f377f3897da877b8b4790552e4f557ceeed78157R126

https://github.com/openedx/edx-enterprise/pull/2121/files#diff-d45bd0ecb65fe9b632f0d331f377f3897da877b8b4790552e4f557ceeed78157R150